### ISSUE TITLE
Ingest Tree view should show relative path and date modified

### ISSUE THREAD

**Issue by cam156** [psu-stewardship/archivesphere#30]
When I view an Ingest/ Accession
For each file in the view
I should see the relative path and date modified in addition to the file type and size

Original story:

As a repository administrator, I would like to see a detailed inventory of the objects in a particular ingest, in order to better appraise the content for purposes of describing the collection or integrating it with description for the corresponding analog material of a collection.

Original Done Criteria:

I can access a collection and see a detailed list/tree structure of its objects, including file name, file path, format, size, last modified.

**Comment by cam156** [comment]
We already have the relative_path as a string for each file
In addition we have the last modified date (which is usually the upload date)

**Comment by cam156** [comment]
Where does this data get included?  The simplest form should be done first and then get feedback.  If the feedback varies greatly we will open a new ticket.

**Comment by hectorcorrea** [comment]
Fixed in 948be7ce65baca745927e571e5bd2760e474fbba 
![screen shot 2015-08-20 at 3 32 31 pm](https://cloud.githubusercontent.com/assets/568286/9393586/b4467192-4750-11e5-8408-4f3d3b5601d4.png)
